cirq/circuits/circuit.py

```diff
@@ -1527,10 +1530,23 @@ def _get_operation_circuit_diagram_info_with_fallback(
 
     return protocols.CircuitDiagramInfo(wire_symbols=symbols)
 
+def _is_valid_identifier(text):
```

Comment:
This is the method I wanted to be called `_is_exposed_formula`

Reply:
Done,